roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1066,36 +1066,40 @@ def get_openshift_version(facts, cli_image=None):
         if 'version' in facts['common'] and facts['common']['version'] is not None:
             return facts['common']['version']
 
-    if os.path.isfile('/usr/bin/openshift'):
-        _, output, _ = module.run_command(['/usr/bin/openshift', 'version'])
-        version = parse_openshift_version(output)
-
+    # If this is a containerized host, we need to get the openshift version
```

Comment:
FWIW your branch actually passed an install without docker there for me. This PR may be unnecessary so I am going to close for now.